.github/workflows/ci.yml

```diff
@@ -29,11 +29,27 @@ jobs:
         env: #TODO: Don't use GHA secrets for this
           DOCKER_HUB_USERNAME: ${{ secrets.DOCKER_HUB_USERNAME }}
           DOCKER_HUB_PASSWORD: ${{ secrets.DOCKER_HUB_PASSWORD }}
+      - id: create_token
```

Comment:
This isn't actually necessary in this setup, but it's actually a nice visual indicator vs the github one